cmake/FXdivConfig.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
+
+# Set and check FXdiv_INCLUDE_DIR
+set_and_check(FXdiv_INCLUDE_DIR "@PACKAGE_FXdiv_INCLUDE_DIR@")
```

Comment:
@ConnorBaker perhaps this doesn't make sense, given that we're not passing this as a variable through `PATH_VARS`.